pxr/imaging/plugin/hdRpr/materialAdapter.cpp

```diff
@@ -376,27 +376,27 @@ void MaterialAdapter::PoulateUsdPreviewSurface(const MaterialParams & params, co
 		}
 		else if (paramName == HdRprTokens->metallic)
 		{
-			m_vec4fRprxParams.insert({ RPRX_UBER_MATERIAL_REFLECTION_METALNESS, VtValToVec4f(paramValue)});
```

Comment:
Is RPRX_UBER_MATERIAL_REFLECTION_METALNESS_MODE set somewhere?

Reply:
Looks like no and no one used this before
RPRX_UBER_MATERIAL_REFLECTION_METALNESS_MODE is not present in RprSupport.h and RadeonProRender.h
Are you sure that such things exist?

Reply:
Sorry RPRX_UBER_MATERIAL_REFLECTION_MODE

Reply:
[here](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/5b12a652c648122c02a162cd2bc9a827bcbc195e/pxr/imaging/plugin/hdRpr/materialAdapter.cpp#L462) and [here](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/5b12a652c648122c02a162cd2bc9a827bcbc195e/pxr/imaging/plugin/hdRpr/materialAdapter.cpp#L474)